docs/markdown/Dependencies.md

```diff
@@ -255,6 +255,13 @@ libraries that have been compiled for single-threaded use instead.
 
 `method` may be `auto`, `config-tool`, `pkg-config`, `cmake` or `extraframework`.
 
+## Fortran Coarrays
+
+Coarrays are a Fortran language intrinsic feature, enabled by 
+`dependency('coarray')`.
```

Comment:
This should mention that this is only available since 0.50.0.

Also, markdown ignores single linefeeds. If you want a chapter break, there need to be two linefeeds. If not, the text should be rewrapped so that there is no extra linefeed here.